src/openzaak/notifications/tests/utils.py

```diff
@@ -35,13 +35,13 @@
 
 class NotificationsConfigMixin:
```

Comment:
This should probably be merged with the mixin from `__init__.py`, and the classmethod to configure should be invoked on `setUpTestData` automatically I think. Only transaction test case then needs special treatment.